doozerlib/cli/release_gen_assembly.py

```diff
@@ -27,12 +30,25 @@ def releases_gen_assembly(ctx, name):
 @click.option('--standard', 'standards', metavar='4.y.z-ARCH', default=[], multiple=True, help='The name and arch of an official release (e.g. 4.8.3-x86_64) where ARCH in [x86_64, s390x, ppc64le, aarch64].')
 @click.option("--custom", default=False, is_flag=True,
               help="If specified, weaker conformance criteria are applied (e.g. a nightly is not required for every arch).")
+@click.option('--in-flight', 'in_flight', metavar='EDGE', help='An in-flight release that can upgrade to this release')
+@click.option('--previous', 'previous_list', metavar='EDGES', default=[], multiple=True, help='A list of releases that can upgrade to this release')
+@click.option('--auto-previous', 'auto_previous', is_flag=True, help='If specified, previous list is calculated from Cincinnati graph')
+@click.option("--graph-url", metavar='GRAPH_URL', required=False,
+              default='https://api.openshift.com/api/upgrades_info/v1/graph',
+              help="When using --auto-previous, set custom Cincinnati graph URL to query")
+@click.option("--graph-content-stable", metavar='JSON_FILE', required=False,
+              help="When using --auto-previous, override content from stable channel - primarily for testing")
+@click.option("--graph-content-candidate", metavar='JSON_FILE', required=False,
+              help="When using --auto-previous, override content from candidate channel - primarily for testing")
+@click.option("--suggestions-url", metavar='SUGGESTIONS_URL', required=False,
+              default="https://raw.githubusercontent.com/openshift/cincinnati-graph-data/master/build-suggestions/",
+              help="When using --auto-previous, set custom suggestions URL, load from {major}-{minor}-{arch}.yaml")
 @pass_runtime
 @click.pass_context
-def gen_assembly_from_releases(ctx, runtime, nightlies, standards, custom):
+def gen_assembly_from_releases(ctx, runtime: Runtime, nightlies: Tuple[str, ...], standards: Tuple[str, ...], custom: bool, in_flight: Optional[str], previous_list: Tuple[str, ...], auto_previous: bool, graph_url: Optional[str], graph_content_stable: Optional[str], graph_content_candidate: Optional[str], suggestions_url: Optional[str]):
```

Comment:
This is a lot of options added to the invocation that in practice will not change. Do we need to have this as variable as this? Maybe we can have them defined in constants.py, so they can be overridden there if we need to experiment during development?

Reply:
I also doubt they will change. I just blindly copied those options and some code from the existing `release:calc-previous` command.

Reply:
I agree with joep, don't see any value in adding them